core/examples/generated/lib/railjson_generator/__init__.py

```diff
@@ -1,7 +1,7 @@
 import sys
 from pathlib import Path
 
-api_schema_parent_path = Path(__file__).parents[5] / "api" / "osrd_infra" 
+api_schema_parent_path = Path(__file__).parents[5] / "python" / "osrd_schemas" 
```

Comment:
Now that schemas are isolated. We should be able to `pip install --dev` it as a library. We can plan to do it in a separate PR though.